### ISSUE TITLE
Application fails to load under Internet Explorer 11

### ISSUE THREAD

**Issue by ryanewtaylor** [brygrill/wab-react-webpack#8]
When I launch the application under Internet Explorer 11 the WAB shows the initial loading screen but soon after fails. The following errors are thrown

> SCRIPT1010: Expected identifier
> MapManager.js (75,1)

> TypeError: Object doesn't support property or method 'getInstance' TypeError: Object doesn't support property or method 'getInstance'
   at _loadMap (https://localhost:3002/jimu.js/LayoutManager.js?wab_dv=2.4:677:7)
   at onAppConfigLoaded (https://localhost:3002/jimu.js/LayoutManager.js?wab_dv=2.4:107:7)
   at Anonymous function (https://js.arcgis.com/3.20/init.js:63:268)
   at c (https://js.arcgis.com/3.20/init.js:119:178)
   at n.emit (https://js.arcgis.com/3.20/init.js:124:374)
   at n.emit (https://js.arcgis.com/3.20/init.js:125:480)
   at e.prototype.emit (https://js.arcgis.com/3.20/init.js:117:290)
   at publish (https://js.arcgis.com/3.20/init.js:171:319)
   at Anonymous function (https://localhost:3002/jimu.js/ConfigManager.js?wab_dv=2.4:131:9)
   at Anonymous function (https://js.arcgis.com/3.20/init.js:63:268)

It's not clear to me why it fails in Internet Explorer at this time. It functions fine in Firefox, Edge, and Chrome. I will update this ticket should I find more info (our team is required to support IE 11).

**Comment by brygrill** [comment]
Yea, sorry I never check IE....that looks like it is throwing before it gets to the widget? I will test it out when I have some time, usually its just a matter of adding the right polyfill.

**Comment by ryanewtaylor** [comment]
Yeah, I believe it's throwing before getting to the widget. It would seem like a polyfill issue. 

Perhaps related, but maybe not. I found that when I `yarn deploy` to create a widget, copy that widget to another WAB application, and run it in IE 11, the widget fails with an `Array.prototype.toString: 'this' is null or undefined`. Ultimately, this may be a core-js issue! 

Our team has found that whenever core-js is a dependency, we need to pin it at core-js 2.4.1. Any other version causes IE 11 to error with the previously mentioned message. To resolve (temporarily) I removed @babel/polyfill from the build and explicitly imported core-js @ 2.4.1. This solved the `Array.prototype.toString` issue, but not the original issue I described. Perhaps it's only tangentially related but the info may be important.

If I find a solution I'd be happy to submit a PR!
